SETUP.md

```diff
@@ -21,6 +22,10 @@ For training at scale, operationalization or hyperparameter tuning, it is recomm
 
 Depending on the type of NLP system and the notebook that needs to be run, there are different computational requirements. Currently, this repository supports **Python CPU** and **Python GPU**. A conda environment YAML file can be generated for either CPU or GPU environments as shown below in the *Dependencies Setup* section.
 
+## Create a cloud-based workstation 
+[Azure Machine Learning service](https://azure.microsoft.com/en-us/services/machine-learning-service/)’s Notebook Virtual Machine (VM), is a cloud-based workstation created specifically for data scientists. Notebook VM based authoring is directly integrated into Azure Machine Learning service, providing a code-first experience for Python developers to conveniently build and deploy models in the workspace. Developers and data scientists can perform every operation supported by the Azure Machine Learning Python SDK using a familiar Jupyter notebook in a secure, enterprise-ready environment. Notebook VM is secure and easy-to-use, preconfigured for machine learning, and fully customizable. 
```

Comment:
one question Heather, how can we install the dependencies in the notebook VM? is there a command line where we can install the conda env, etc?

Reply:
There is a terminal that allows you to walk through the same steps

Reply:
I updated the wording to be slightly more clear to do it in the terminal.